trpc/router/mailbox/procedure.ts

```
@@ -1,12 +1,12 @@
 import { TRPCError } from "@trpc/server";
 import { z } from "zod";
-import { getMailboxBySlug } from "@/lib/data/mailbox";
+import { getMailbox } from "@/lib/data/mailbox";
 import { protectedProcedure } from "@/trpc/trpc";
 
 export const mailboxProcedure = protectedProcedure
   .input(z.object({ mailboxSlug: z.string() }))
   .use(async ({ ctx, input, next }) => {
```

ESLint

[error] 'input' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)